unsafe/src/main/java/org/apache/spark/unsafe/map/BytesToBytesMap.java

```diff
@@ -291,6 +308,14 @@ Location with(int pos, int keyHashcode, boolean isDefined) {
       return this;
     }
 
+    Location with(long fullKeyAddress, boolean isDefined) {
```

Comment:
This method is only ever called when `isDefined` is true, so I think we can just omit that argument.